tests/Aspire.Dashboard.Tests/MockConfiguration.cs

```diff
@@ -0,0 +1,40 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Primitives;
+
+namespace Aspire;
+
+internal sealed class MockConfiguration : IConfiguration, IEnumerable
```

Comment:
You don't have to write a mock, there's an easy way to spin up config over a dictionary.

```C#

var config = new ConfigurationManager();
config.AddInMemoryCollection(new Dictionary<string,string?>{

});
```

Reply:
Nice. Updated.